pontoon/insights/tasks.py

```diff
@@ -438,8 +460,22 @@ def get_time_to_review_data(category, activities, locale):
     return sum(times_to_review, timedelta()) / len(times_to_review)
 
 
-def query_activity_actions(start_of_today):
-    """Get actions of the previous day, needed for the Translation and Review activity charts."""
+def get_chrf_score(action):
+    try:
+        approved_translation = Translation.objects.get(
+            entity=action["translation__entity"],
+            locale=action["translation__locale"],
+            approved=True,
+        ).string
```

Comment:
It seems pretty inefficient to look up each approved translation individually. It's fine for the single-run migration, but couldn't all of the rejected pretranslations' approved versions translations be looked up at once?